Development/src/components/URLField.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Typography } from '@material-ui/core';
+import get from 'lodash/get';
+
+const UrlField = ({ source, record = {} }) => (
```

Comment:
UrlField or URLField? Should the filename and component name match?
And be consistent with `TAIField` (or `TaiField` if we're going that way...).

Reply:
The name UrlField was used as the component was a drop in replacement for the react-admin version, exception being it themes correctly. I will change to URLField to be more consistent.

Reply:
Let's discuss. Not sure whether reusing names is frowned upon in React? If not, maybe we should be going the other way and making it TaiField...